src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts

```diff
@@ -181,3 +182,37 @@ const renderStylus = ({ filename, root, paths }): Promise<string> => {
         })
     );
 };
+
+export interface ForEachStylesheetFn {
+  (css: string, filePath: string, entryPoint: NgEntryPoint): string | Promise<string>;
+}
+
+export function forEachStylesheet(fn: ForEachStylesheetFn): Transform {
```

Comment:
We need a test here that verifies the correct behaviour.

- [ ] create a mock graph and feed some nodes to the transform
- [ ] verify that `fn` is called for every stylesheet
- [ ] verify that result from `fn` is applied to stylesheet nodes